File: tests/dynamic_fixtures/php/xss_positive.php
<?php
// XSS — positive fixture.
// Vulnerable: echoes raw user input into HTML output without escaping.
// Entry: renderPage($userInput)  Cap: HTML_ESCAPE
// Expected verdict: Confirmed (<script>NYX_XSS_CONFIRMED</script> echoed)

function renderPage($userInput) {
    echo "__NYX_SINK_HIT__\n";
    echo '<html><body>' . $userInput . '</body></html>' . "\n";
}

File: tests/dynamic_fixtures/go/xss_positive.go
// XSS — positive fixture.
// Vulnerable: echoes raw user input into HTML output without escaping.
// Entry: RenderPage(userInput string)  Cap: HTML_ESCAPE
// Expected verdict: Confirmed (<script>NYX_XSS_CONFIRMED</script> echoed)

package entry

import "fmt"

func RenderPage(userInput string) {
	fmt.Print("__NYX_SINK_HIT__\n")
	fmt.Print("<html><body>" + userInput + "</body></html>\n")
}
